src/clarity-angular/buttons/_buttons.clarity.scss

```diff
@@ -215,6 +241,64 @@ $clr-button-appearance-map: (
         @include generateButton();
         @include clr-getStylesFromMap($clr-button-appearance-map, form);
         @include populateButtonProperties(default);
+    }
+
+    .btn-group {
+        //.btn-outline-<state> is the new BS4 naming convention
+        //TODO: we support both the previous one and the new one but we will deprecate the old ones from the example in 0.5.0
+        &.btn-secondary > .btn,
```

Comment:
I discussed this with @youdz during a code pair and he mentioned that we should change our class names at one go when we decide to adopt the new naming conventions. So thats why I used to old convention that we have for buttons too. @youdz is that right?